Networking/Networking/Mapper/ProductVariationMapper.swift

```diff
@@ -41,3 +68,27 @@ private struct ProductVariationEnvelope: Decodable {
         case productVariation = "data"
     }
 }
+
+/// ProductVariationsEnvelope Disposable Entity
+///
+/// `Variations/batch` endpoint returns the requested update product variations document in a `update` key, nested in a `data` key.
+/// This entity allows us to do parse all the things with JSONDecoder.
+///
+private struct ProductVariationsEnvelope: Decodable {
+    let updatedProductVariations: [ProductVariation]
+
+    private enum CodingKeys: String, CodingKey {
+        case update = "update"
+    }
+
+    private enum TopLevelCodingKeys: String, CodingKey {
+        case data = "data"
```

Comment:
nit: I think you can omit the explicit value.

Reply:
Yes, I updated this 👍